### DISCUSSION TITLE
Change default "original file" folder

### DISCUSSION THREAD

**Discussion by thebyronq** [hsoft/dupeguru#994] [Q&A]
Hi all,
Just curious to know if there is a way to change the default folder for original files?
As I am wanting to scan two network locations, however it is defaulting to the duplicate folder, with the original folder as the "duplicates"
Thanks in advance!

**Comment by Mewto7** [comment]
I am not an expert so take this with caution. Do you mean you want to change the reference folder? If so, just click on normal and set it to reference. Set the original folder as reference and the one you want to dedupe as normal and that should do it.
